recursio-client/Managers/CharacterManager.gd

```diff
@@ -56,11 +57,9 @@ func _ready():
 	
 	_error = Server.connect("game_result", self, "_on_game_result")
 
-
 	_player_rpc_id = get_tree().get_network_unique_id()
 	set_physics_process(false)
 
-
```

Comment:
Just fyi, in GDScript normally two new lines are between functions (at least Karl said so)

Reply:
Oh sorry, when I remember to do it I always try to keep only one line between them. Will use two from now on.